Cargo.lock

```diff
@@ -1988,7 +2047,25 @@ dependencies = [
  "bytes",
  "ed25519-dalek",
  "hex",
- "k256",
+ "k256 0.11.6",
```

Comment:
You guys may also want to bump K256 & related crates, here's the ethers PR with some indications on the breakages https://github.com/gakonst/ethers-rs/pull/2260